torch/_dynamo/variables/torch.py

```diff
@@ -496,6 +496,10 @@ def get_state_from_generator():
             return TorchVariable(torch.add, **options).call_function(
                 tx, [args[0], result], {}
             )
+        elif self.value == torch.nn.init._calculate_correct_fan:
+            return UserFunctionVariable(
+                torch.nn.init._calculate_correct_fan, **options
+            ).call_function(tx, args, {})
```

Comment:
This helped us to prove the idea that we can use inlining as the ultimate fallback for ```call_function```, we will do code refactor to remove redundant code later. Thanks!